tsl/test/sql/CMakeLists.txt

```diff
@@ -105,6 +104,11 @@ if((${PG_VERSION_MAJOR} GREATER_EQUAL "14"))
   endif()
 endif()
 
+# Skip this test on PG version 15
+if((${PG_VERSION_MAJOR} LESS "15"))
+  list(APPEND TEST_FILES cagg_errors_deprecated.sql)
+endif()
```

Comment:
Same here... adding the DEBUG flag check do we still need this change??

Reply:
Currently this tests runs on both Debug and Release builds. Thus iam skipping this tests on PG15.

Reply:
This change is not necessary anymore (same as other tests)